packages/pds-ember/app/styles/pds/components/button/_config.scss

```diff
@@ -5,7 +5,7 @@ $_module: "PDS.Components.Button";
 @use "../../tokens/color";
 @use "../../tokens/font";
 @use "../../tokens/scale";
-@use "../../tokens/typography" as Typography;
+@use "../../core/typography/config" as Typography;
```

Comment:
Typography mixins moved into `core/typography/_config.scss` to stay with the rest of typography styles.

**Why `core/typography`?**
Typography isn't a component per se, nor is it a set of tokens or utils. It's a set of foundational/core styling that applies to many different components, and affects how the vast majority of styles are defined.

Reply:
Can you `@forward` config from that file so that it's not a breaking change, and maybe add a `@warn` too?

Reply:
(no idea how frequently it's used elsewhere, but it'd be good to start developing these patterns to keep breaking changes to a minimum)

Reply:
Even if I forward the config, not all mixins will be available in the module, because the `PDS.Tokens.Typography@Code` mixin is now the `PDS.Components.CodeBlock@appearance` mixin.  Sass also doesn't support selective forwarding (its either all or nothing), so the amount of work to retain the API isn't worth the effort, because it requires:

1. `@forward "core/typography/config";` (handles most of the previously available mixins)
2. define a deprecated `PDS.Tokens.Typography@Code` mixin in order to wrap the `PDS.Components.CodeBlock@appearance` mixin (which doesn't seem worth the effort, given that the old mixin isn't being used in HCP anyway)

## Regarding Stability
As we move forward with additional NPM releases, we need to have clear guidelines on what constitutes a breaking change in the context of public vs private APIs.  IMO, it's more important to avoid breaking changes in the `PDS.Theme` module (which still needs a lot of work to stabilize).  However, we can't ignore that consumers may still consume the lower-level modules directly, so we need to document where the line between public and private APIs/modules lies and make sure that consumers are aware that private APIs can change at any time.

If you have any thoughts about this, please hit me up on Slack so that we can iron out some initial guidelines.

## Supplemental
Doing a quick search in HCP, it looks like none of the mixins are being used outside of the PDS addon.  Given that there are only 4 places in the HCP codebase that reference this module (all within the old PDS addon), I'm confident that this breaking change shouldn't have any adverse affects on HCP UIs.

![CleanShot 2020-10-12 at 18 27 36](https://user-images.githubusercontent.com/545605/95798840-9c41b600-0cb8-11eb-9e51-bb89ddedf301.png)